magellan-library/src/main/java/com/wealthfront/magellan/view/ViewExtensions.kt

```diff
@@ -3,7 +3,7 @@ package com.wealthfront.magellan.view
 import android.view.View
 import android.view.ViewTreeObserver
 
-internal fun View.whenMeasured(onMeasured: () -> Unit) {
+fun View.whenMeasured(onMeasured: () -> Unit) {
```

Comment:
We probably shouldn't make this public, I think there's a competing method in ktx. Let's double check that before making this public

Reply:
d